docs/userguide/first_steps.rst

```diff
@@ -11,10 +11,14 @@ Welcome to Parsec! Here's a brief overview of the user interface.
     :align: center
     :alt: Parsec graphical user interface overview
 
-\1. **Organization section**: select your organization and access your organization details (if allowed by your profile).
+\1.**Organization section**: select your organization.
```

Comment:
```suggestion
\1. **Organization section**: select your organization.
```

A space is needed, otherwise it does not render properly
<img width="843" height="306" alt="image" src="https://github.com/user-attachments/assets/bae18082-38b6-4632-b5e0-287e4a2b26ee" />